package.json

```diff
@@ -268,6 +268,10 @@
                 "command": "language-julia.activateFromDir",
                 "title": "Julia: Activate Parent Environment"
             },
+            {
+                "command": "language-julia.clearDiagnostics",
```

Comment:
The "user" terminology can be ambiguous I think, because it's not users who provide diagnostics but it's a provider package. Maybe we can rename it as `clearCustomDiagnostics` and it will make it more explicit that it's different from the diagnostics provided by LS ?

Reply:
Right, the command really should be named `language-julia.clearRuntimeDiagnostics`.